src/Symfony/Component/Messenger/RoutableMessageBus.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger;
+
+use Psr\Container\ContainerInterface;
+use Symfony\Component\Messenger\Exception\InvalidArgumentException;
+use Symfony\Component\Messenger\Stamp\BusNameStamp;
+
+/**
+ * Bus of buses that is routable using a BusNameStamp.
+ *
+ * This is useful when passed to Worker: messages received
+ * from the transport can be sent to the correct bus.
+ *
+ * @experimental in Symfony 4.2
```

Comment:
`@experimental in 4.3` (omitting `Symfony` for consistency), same for other classes added here

Reply:
Just changed all occurrences, great catch 👍